languages/wp-document-revisions.pot

```diff
@@ -59,314 +59,314 @@ msgstr ""
 msgid "Document draft updated. <a target=\"_blank\" href=\"%s\">Download document</a>"
 msgstr ""
 
-#: includes/class-wp-document-revisions-admin.php:232
```

Comment:
No harm in updating the POT file in this PR, but for future PRs, if you'd prefer, I added a GitHub Action to generate a new POT file after a PR is merged.

Reply:
FWIW, since I have a clone of the plugin, it seems that the Generate POT is run as a GIT action with every commit (and I have been doing quite a number trying to understand how to get the tests through!)
So I had an outstanding Pull Request in my code for the POT file.
After I had generated the Pull request to you, I noticed that I had an outstanding Pull request in my version. So to minimise the differences going forward, I merged in the Pullrequest on my side. I then has been propagated as part of this Pull request.
I would be very grateful if you were able to change the Action to be after a merge rather than a commit.